src/js/modules/ContactHours.js

```diff
@@ -33,50 +36,42 @@ const i18nMessagesWeekdayMap = {
     />,
 };
 
-class Hours extends Component {
+class ContactHours extends Component {
 
-  sort() {
+  sort(hours) {
     // TODO: Joplin data MUST include data for all 7 days of week.
     let now = new Date()
-    let sorted = sortBy(this.props.hours, ['dayOfWeekNumeric']);
+    let sorted = sortBy(hours, ['dayOfWeekNumeric']);
     let index = findIndex(sorted, {'dayOfWeek': now.getDay()});
     return sorted.splice(index).concat(sorted);
   }
 
   render() {
-    const hours = this.sort();
-    let JSX;
-
-    if (!hours || !hours.length) {
-      JSX = null;
-    } else {
-      JSX = (
-        <div className="coa-section__map">
-          <h5>Hours</h5>
-          <table className="usa-table-borderless">
-            <thead className="usa-sr-only">
-              <tr>
-                <th scope="col">Day</th>
-                <th scope="col">Open - Close Hours</th>
+    const hours = this.sort(this.props.hours);
+    return (
+      <div className={`${this.props.className} coa-ContactHours`}>
+        <AirplaneSVG size="20"/>
+        <table className="usa-table-borderless">
```

Comment:
Should we copy/🍝  these two classes from USWDS? You're doing a lot of work moving us off USWDS so just wondering why you didn't here too.

`usa-table-borderless`: https://github.com/uswds/uswds/blob/9ff31e9e2c8906246b79d233116ef842cbc5a52c/src/stylesheets/elements/_table.scss#L29
`usa-sr-only`: https://github.com/uswds/uswds/blob/1b16c2dc97c3a38c91f9af1eb5c2694c6d77e773/src/stylesheets/core/_utilities.scss#L52-L69

Reply:
I agree we should, I mostly didn't want to deal with it the later it got in the day. We're probably leaning on a lot of their base styles too -- I'll log this in an issue.